applications/base/skills.py

```
@@ -507,16 +973,24 @@ class check_pose_implementation(BaseTool):
     return_direct: ClassVar[bool] = False
     verbose: ClassVar[bool] = False
 
-    def _run(self, object: str = None, cam_name: str = "camera", parent_frame: str = "base_link", gripper_frame: str = None) -> list:
+    def _run(self, object: Optional[str] = None, camera_name: str = "camera", parent_frame: str = "base_link", gripper_frame: str = None) -> Optional[list]:
```

Ruff (0.12.2)

Unused method argument: `gripper_frame`

(ARG002)